.github/workflows/deploy_go-pkgs.yml

```diff
@@ -19,6 +19,9 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
+# Allows you to run this workflow manually from the Actions tab
+workflow_dispatch:
```

Comment:
There's already a `workflow_dispatch` above, it should not be here...

Reply:
Not sure how that got in there. Fixed.